### ISSUE TITLE
"Open in new private window" text missing bookmark context menu

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#15046]
## Description 

A Turkish user is reporting  not seeing the text for the `Open in private window` in the context menu when right-clicking on a bookmark. Note that selecting the blank space still does open the bookmark in a private window, it’s just that the text is missing. User has browser text in Turkish, which may or may not be part of the issue (translation).
![image](https://user-images.githubusercontent.com/3299098/113176028-eb48bc80-9200-11eb-9ec6-79998d912f50.png)

## Reproduces how often: 

Always

## Brave version (brave://version info)

Version 1.22.70 Chromium: 89.0.4389.105 (Official Build) (x86_64)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? N/A
- Can you reproduce this issue with the nightly channel? N/A

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

## Miscellaneous Information:

Original user thread: https://community.brave.com/t/open-bookmarks-in-new-private-tab-vanished/224756/3

**Comment by mkarolin** [comment]
QA STR:
1. Start browser
2. Enable showing of the bookmarks bar
3. Add at least 2 bookmarks 
4. Right-click on an empty space in the bookmarks bar.

**Comment by kjozwiak** [comment]
@mkarolin can we get this uplifted into `1.23.x`?

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.24.46 Chromium: 90.0.4430.41 (Official Build) nightly (x86_64)
-- | --
Revision | e9c92b1eaca8487e212f3f6bc081fdb6d4863759-refs/branch-heads/4430@{#723}
OS | macOS Version 10.15.7 (Build 19H524)
```

Reproduced the issue using Turkish language in `1.22.71` and STR from https://github.com/brave/brave-browser/issues/15046#issuecomment-811986531.

Confirmed fixed using same STR and `1.24.46`.

`1.22.71` | `1.24.46`
----- | -----
<img width="1199" alt="1 22 71" src="https://user-images.githubusercontent.com/28145373/113572410-27579500-95e6-11eb-9fe5-d09af42f498d.png"> | <img width="1197" alt="1 24 46" src="https://user-images.githubusercontent.com/28145373/113572412-29215880-95e6-11eb-8e9a-3e98d82e0055.png">
